b2sdk/transfer/outbound/copy_manager.py

```diff
@@ -32,38 +31,15 @@ class CopyManager(metaclass=B2TraceMetaAbstract):
     def __init__(self, services, max_copy_workers=10):
```

Comment:
do we need the parameters in all Manager classes have different names for the worker count? Maybe it could all be `max_workers`, then maybe more of `__init__` could go up in class hierarchy?

Reply:
:thinking: May be a good idea.

Reply:
if we are refactoring, lets refactor